gluten-ut/spark33/src/test/scala/org/apache/spark/sql/execution/GlutenSQLAggregateFunctionSuite.scala

```diff
@@ -33,4 +34,29 @@ class GlutenSQLAggregateFunctionSuite extends GlutenSQLTestsTrait {
     checkAnswer(df, Seq(Row(3, 5)))
     assert(getExecutedPlan(df).count(_.isInstanceOf[HashAggregateExecBaseTransformer]) == 4)
   }
+
+  testGluten("VELOX-12542: Return NaN or null when dividing by zero") {
```

Comment:
Perhaps drop the prefix 'VELOX-12542' as the test is shared between Velox and CH backend.